src/main/java/core/basesyntax/tools/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.tools;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int colorRandomIndex = new Random().nextInt(Color.values().length);

```

Comment:
Overloaded expression, please introduce some variables. Also Random should be class level variable